api-ref/source/dedicated/hm.rst

```diff
@@ -16,6 +16,7 @@ Constraints
 The security groups must have rules that allow access from 100.125.0.0/16. If
 you want to use UDP for health checks, ensure that the protocol of the backend
 server group is UDP.
+Error TYPO INSERTION
```

Comment:
This is completely wrong, please remove it

Reply:
fixed